drivers/i2c/i2c_mchp_xec.c

```diff
@@ -22,6 +22,9 @@
 /* 25 us */
 #define MAX_CLK_STRETCHING  5
 
+uint8_t s_port_sel;
```

Comment:
Make these `static` so they don't leak into the global namespace.  But shouldn't they be in a device-specific structure?  Or are these just here for diagnostics during development and will go away when the `printk`s are removed?

Reply:
These are here only for diagnostics. This will get into the device specific structure as this should be done for each i2c controller.